### PR TITLE
built store: cart

### PR DESCRIPTION
**Pull request by HelloLisaNYC**
built cart in redux store

### CODE REVIEW

File: client/store/cart.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+const SET_CART = 'SET_CART';
+const UPDATED_CART = 'UPDATED_CART';
```
Comment (vljones133) [inline client/store/cart.js:4]: Update_cart

File: client/store/cart.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const setCart = (cart) => {
+  return {
+    type: SET_CART,
+    cart,
+  };
+};
+
+const gotUpdatedCart = (updatedCart) => {
```
Comment (vljones133) [inline client/store/cart.js:13]: updateCart

File: client/store/cart.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  return {
+    type: UPDATED_CART,
```
Comment (vljones133) [inline client/store/cart.js:15]: tense

File: client/store/cart.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+export const updateCart = (productId, quantity) => {
+  const cartUpdate = { productId, quantity };
+  const token = window.localStorage.getItem('token');
+  return async (dispatch) => {
+    const { data: updatedCart } = await axios.post('/api/cart', cartUpdate, {
+      headers: {
+        authorization: token,
+      },
+    });
+    dispatch(gotUpdatedCart(updatedCart));
```
Comment (vljones133) [inline client/store/cart.js:41]: will be changed here

File: client/store/cart.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+export const removeProductFromCart = (productId) => {
+  const token = window.localStorage.getItem('token');
+  return async (dispatch) => {
+    const { data: updatedCart } = await axios.delete(`/api/cart/${productId}`, {
+      headers: {
+        authorization: token,
+      },
+    });
+    dispatch(gotUpdatedCart(updatedCart));
```
Comment (vljones133) [inline client/store/cart.js:53]: and here

File: client/store/cart.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+      },
+    });
+  };
+};
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case SET_CART:
+      return action.cart;
+    case UPDATED_CART:
```
Comment (vljones133) [inline client/store/cart.js:72]: tense

**Review by vljones133 — CHANGES_REQUESTED** [review]

File: client/store/cart.js (hunk 6 above)
Comment (selgoobler) [reply client/store/cart.js:72]: looks good so far, did you want me to approve now?

**Review by vljones133 — APPROVED** [review]

**Pull request merged**